src/website/app/demos/Popover/bestPractices.js

```diff
@@ -175,19 +175,19 @@ const BadContent = (
 export default [
   {
     type: 'do',
-    title: 'use Popover to provide help to the user',
-    example: <PopOverDo />,
-    description: 'Provide extra context for selected interface elements.'
+    description:
+      'Use Popover to provide extra context for selected interface elements.',
+    example: <PopOverDo />
   },
   {
     type: 'dont',
-    title: 'add too much content',
+    description: `Don't place long content in a Popover. If more context is
+needed, provide a link to another resource with more detail. Add a \`max-width\`
```

Comment:
it is unclear to me whether "Add a max-width..." is something that users should or shouldn't do